app/models/admin.rb

```diff
@@ -1,6 +1,7 @@
 class Admin < ActiveRecord::Base
 
   require 'securerandom'
+  include ActiveModel::Dirty
```

Comment:
no need to include `ActiveModel::Dirty` since `ActiveRecord::Base` already has all functionality provided by `ActiveModel`.